asterics-web-vue/src/components/documentation/DocElement.vue

```diff
@@ -0,0 +1,77 @@
+<template>
```

Comment:
Well, I'm pretty sure, that the first image was made on an earlier branch, probably `sprint201808-doc`.

The element DocElement is used in the DocSidebar, and it uses some features that are rather discreet and won't be seen at first sight (see 2).

(1) I'll recommend you to install Vue Debugger Tools for Chrome. This way you'll find components like in this case, typically quickly.

![image](https://user-images.githubusercontent.com/13609768/45306027-e83f3100-b51b-11e8-91e2-4249a4475cb4.png)


(2) The component `DocSidebar` is actually a recursive component (cf. [here](https://vuejs.org/v2/guide/components-edge-cases.html#Recursive-Components)).
I've renamed the component unintentionally to `doc` in the source code. It should be more comprehensible now:

BEFORE:
```html
      .....
      <div v-for="i in element.children" :key="i.sha">
        <doc :element="i" :tag="tag"/>
      </div>
    </v-list-group>
  </div>
</template>
```
```javascript
import { mapGetters } from "vuex";

export default {
  name: "doc", // recursively used component <doc/>
  computed: {
```

NOW:
```html
      .....
      <div v-for="i in element.children" :key="i.sha">
        <doc-element :element="i" :tag="tag"/>
      </div>
    </v-list-group>
  </div>
</template>
```
```javascript
import { mapGetters } from "vuex";

export default {
  name: "doc-element",
  computed: {
```

Reply:
ah, ok, thx. Then I indeed looked at the wrong version, I think it was the branch `sprint201808-doc`.
Yes, `doc-element` is better! Maybe `doc-sidebar-element` would be even better because then it would be very clear where this element is used. However it is also fine as it is.